_posts/2018-12-06-pmreorder-basics.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Pmreorder basics
+author: wlemkows
+layout: post
+identifier: pmreorder-basics
+---
+
+
+### Introduction
+
+
+When you write your persistent memory application the good practice is
+to check it under pmemcheck - the tool which we have already described in two
+blog posts some time ago. Feel free to come back to them and remind details
+about [basics usage] [pmemcheck1Link] and [transaction][pmemcheck2Link].
+
+In this post, I am going to introduce another tool for persistence correctness check.
+As you already know, pmemcheck verify if all stores are made persistent in a proper
```

Comment:
As you might already know if you've read the posts linked above, pmemcheck verifies...